src/components/GFERequestPanel.js

```diff
@@ -1333,7 +1331,24 @@ class GFERequestBox extends Component {
             })
         }
         if (valid) {
-            const newId = this.state.careTeamList.length + 1;
+            let newId = this.state.careTeamList.length + 1;
+
+            //TODO: how to account for when you delete item 1 out of 2 items, how do you adjust the id's?
```

Comment:
Is this something still needed to be addressed?

Reply:
No, will remove comment.